patches/01-ocf-mbedtls-config-x509san-anon-psk.patch

```diff
@@ -711,7 +741,17 @@ index 87b4e9192..0223dce8f 100644
  
  /**
   * \def MBEDTLS_ECDH_C
-@@ -2779,7 +2806,9 @@
+@@ -2762,7 +2795,9 @@
+  *
+  * Requires: MBEDTLS_ECP_C
+  */
++#ifdef OC_PKI
+ #define MBEDTLS_ECDH_C
```

Comment:
ECDH doesn't depend on PKI

Reply:
Requires `MBEDTLS_ECP_C`, `MBEDTLS_ECP_C` enabled

Reply:
I've enabled/disabled macro purely to make it compile, so please check if the setup makes sense.

Reply:
MBEDTLS_ECP_C doesn't depends on PKI

Reply:
Yes, I've removed that `ifdef`. So newly ifdef-ed defines for OC_PKI in this PR are:

```
MBEDTLS_KEY_EXCHANGE_ECDHE_RSA_ENABLED
MBEDTLS_ASN1_PARSE_C
MBEDTLS_PK_C
MBEDTLS_PK_PARSE_C
MBEDTLS_RSA_C
MBEDTLS_X509_USE_C
MBEDTLS_X509_CRT_PARSE_C
```

Reply:
👍